File: Day-2/Practice-Set/BankAccount.java
// Create a BankAccount class with accountHolder and balance.
// Print details.

public class BankAccount {
    String accountHolder;
    int balance;

    BankAccount(){
        accountHolder = "Unkown";
        balance = 0;
    }

    public static void main(String[] args) {
        BankAccount c1 = new BankAccount();
        BankAccount c2 = new BankAccount();

        c1.accountHolder = "Yash";
        c1.balance = 50000;

        c2.accountHolder = "Lara";
        c2.balance = 40000;

        System.out.println("Account Holder Name: " + c1.accountHolder);
        System.out.println("Account Balance: " + c1.balance + "\n");
        
        System.out.println("Account Holder Name: " + c2.accountHolder);
        System.out.println("Account Balance: " + c2.balance);
    }
}

File: Day-3/Lab/BankAccount.py
'''
Class: BankAccount

Instance variables:
accountHolder
balance

Static/Class variable:
bankName = "SBI"

Create 3 accounts and print:
Account holder
Bank name
Total accounts created
'''

class BankAccount:
    bankName = "SBI"
    count = 0

    def __init__(self, accountHolder, balance):
        self.accountHolder = accountHolder
        self.balance = balance
        BankAccount.count += 1


acc1 = BankAccount("Yash", 1000)
acc2 = BankAccount("Lara", 1500)
acc3 = BankAccount("Pavii", 2000)

print(f"Account Holder: {acc1.accountHolder}")
print(f"Balance: {acc1.balance}")
print(f"Bank Name: {BankAccount.bankName} \n")

print(f"Account Holder: {acc2.accountHolder}")
print(f"Balance: {acc2.balance}")
print(f"Bank Name: {BankAccount.bankName} \n")

print(f"Account Holder: {acc3.accountHolder}")
print(f"Balance: {acc3.balance}")
print(f"Bank Name: {BankAccount.bankName} \n")

print(f"Total Bank Accounts: {BankAccount.count}")